samples/cpp/prompt_lookup_decoding_lm/prompt_lookup_decoding_lm.cpp

```diff
@@ -9,9 +9,32 @@ namespace {
 
 // only batch_size = 1 currently supported
 constexpr size_t BATCH_SIZE = 1;
-// sequence length axis in key/values tensors, for most cases [BATCH_SIZE, num_kv_heads, seq_len, head_size],
-// threfore usually SEQ_LEN_AXIS = 2
-constexpr size_t SEQ_LEN_AXIS = 2;
+
+size_t get_seq_len_axis(std::shared_ptr<ov::Model> model) {
+    // sequence length axis in key/values tensors, for most cases [BATCH_SIZE, num_kv_heads, seq_len, head_size],
+    // threfore usually seq_length_axis = 2
+    size_t seq_length_axis = 2;
+
+    for (const auto op : model->get_ordered_ops()) {
+        // "ReadValue" node is KV cache representation in stateful model
+        if (std::string("ReadValue") != op->get_type_name()) {
```

Comment:
```suggestion
        if (std::string("ReadValue") != op->get_type_name()) {
```
you can compare ov::ReadValue::get_type_info with op type info.

In this case we will not have magic strings / constants like "ReadValue"

Reply:
I couldn't find `ov::ReadValue::get_type_info` symbol. Is `ov::op::v6::ReadValue::get_type_info_static().name` ok to rely on?

Reply:
maybe we need to compare get_type_info_static for op and class? Because name can be the same, but versions of operations are different

Reply:
I thinks it's ok to skip version check as we don't use ops version specific API. The possible issue would be if name would change with ops version update. Is there a better way to find kv cache nodes?